classes/01class/exercises/c01-git02/alanlima/c01-git02-2.txt

```diff
@@ -0,0 +1,18 @@
+# How to compare two different local repositories using git diff? How about remote ones, can you do the same?
+
```

Comment:
the answer is right but maybe also add a command for comparing two local branches?

Reply:
Thanks @caiocezart, I just added more details how to do that with an example 😁